Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -98,6 +124,12 @@ namespace gdjs {
     let _lobbyChangeHostRequestInitiatedAt: number | null = null;
     let _isChangingHost = false;
     let _lobbyNewHostPickedAt: number | null = null;
+    let _shouldJoinGameRightAfterJoiningLobby = false;
+    let _shouldStartGameRightAfterJoiningLobby = false;
+    let _shouldOpenLobbyPageRightAfterJoiningLobby = false;
+    let _isJoiningOrStartingAGame = false;
```

Comment:
```suggestion
    let _isQuickJoiningOrStartingAGame = false;
```

(as this is just set to true for quick join, not when you're joining or starting a game using the lobby UI)